test/test_grammars.rb

```diff
@@ -43,6 +43,7 @@ class TestGrammars < Minitest::Test
     "82c356d6ecb143a8a20e1658b0d6a2d77ea8126f", # idl.tmbundle
     "9dafd4e2a79cb13a6793b93877a254bc4d351e74", # sublime-text-ox
     "8e111741d97ba2e27b3d18a309d426b4a37e604f", # sublime-varnish
+    "23d2538e33ce62d58abda2c039364b92f64ea6bc", # sublime-angelscript
```

Comment:
I'm surprised this is needed since Licensee seems to already correctly recognize the license on the repository: https://github.com/wronex/sublime-angelscript (right side of the top bar).

Reply:
Here's the travis test that failed before this was added: https://travis-ci.org/github/linguist/builds/283335180?utm_source=github_status&utm_medium=notification

Indeed then, it also recognized the license correctly. Maybe @lildude knows more?

Reply:
Okay, it looks like [Licensee](https://github.com/benbalter/licensee/releases/tag/v8.4.0) does not recognize the license for that repository since v8.4.0. I think we're fine with adding the license hash for now and I'll open an issue on the Licensee repository to investigate.